pkgs/development/python-modules/trafilatura/default.nix

```diff
@@ -48,6 +53,13 @@ buildPythonPackage rec {
   nativeCheckInputs = [ pytestCheckHook ];
 
   disabledTests = [
+    # TypeError: argument of type 'NoneType' is not iterable
+    "test_external"
+    "test_extract"
+
+    # AttributeError: 'NoneType' object has no attribute 'find'
+    "test_table_processing"
```

Comment:
Can we report these issues upstream and link to them?

Reply:
Done! I added the links in the derivations (and in the commit descriptions too).